packages/combo-box/test/scrolling.test.js

```diff
@@ -95,13 +95,13 @@ describe('scrolling', () => {
 
     it('should make selected item visible after reopen', async () => {
       comboBox.open();
-      await aTimeout(0);
+      await nextFrame();
 
       comboBox.value = comboBox.items[50];
       comboBox.close();
 
       comboBox.open();
-      await aTimeout(0);
+      await nextFrame();
```

Comment:
This test was failing on master also for some reason. Changed the timings in the test to make it pass.